target/start-mailserver.sh

```diff
@@ -1064,6 +1065,20 @@ function _setup_elk_forwarder() {
 		> /etc/filebeat/filebeat.yml
 }
 
+function _setup_daily_summery() {
+	notify 'task' 'Setting up daily summery'
+
+	DAILY_SUMM_EMAIL=${DAILY_SUMM_EMAIL:="0"}
+	if [[ ! "$DAILY_SUMM_EMAIL" == 0 ]]; then
+		notify 'inf' "Enable daily mail summery with recipient $DAILY_SUMM_EMAIL"
+
+		sed -i -r 's/HOSTNAME/'$HOSTNAME'/g' /usr/local/bin/dailysummery
+		sed -i -r 's/DAILY_SUMM_EMAIL/'$DAILY_SUMM_EMAIL'/g' /usr/local/bin/dailysummery
+
+		echo "/usr/local/bin/dailysummery" > /etc/cron.daily/dailysummery
```

Comment:
In the daily directory should contain executable scripts?
Just to confirm.

Reply:
due to using logrotate this is obsolet